common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2558,6 +2568,34 @@ func (d *cassandraPersistence) CompleteTasksLessThan(request *p.CompleteTasksLes
 	return p.UnknownNumRowsAffected, nil
 }
 
+func (d *cassandraPersistence) GetTimerTask(request *p.GetTimerTaskRequest) (*p.GetTimerTaskResponse, error) {
+	shardID := d.shardID
+	taskID := request.TaskID
+	visibilityTs := request.VisibilityTimestamp
+	query := d.session.Query(templateGetTimerTaskQuery,
+		shardID,
+		rowTypeTimerTask,
+		rowTypeTimerNamespaceID,
+		rowTypeTimerWorkflowID,
+		rowTypeTimerRunID,
+		visibilityTs,
+		taskID)
```

Comment:
Can this work if I don't have taskID?

Reply:
This method requires taskId to work

There is another method called GetTimerIndexTasks that accepts `minTimestamp`, `maxTimestamp` instead of taskId and returns multiple rows